src/ui/Tooltip.js

```diff
@@ -0,0 +1,123 @@
+﻿import { createEl } from 'core/util/dom';
+import { isString } from 'core/util';
+import { Geometry } from 'geometry';
+import UIComponent from './UI';
+
+
+/**
+ * @property {Object} options
+ * @property {Boolean} [options.autoPan=true]  - set it to false if you don't want the map to do panning animation to fit the opened window.
+ * @property {Number}  [options.width=300]     - default width
+ * @property {Number}  [options.minHeight=120] - minimun height
+ * @property {Boolean} [options.custom=false]  - set it to true if you want a customized tooltip, customized html codes or a HTMLElement is set to content.
+ * @property {String}  [options.title=null]    - title of the tooltip.
+ * @property {String|HTMLElement}  options.content - content of the tooltip.
+ * @memberOf ui.InfoWindow
+ * @instance
+ */
+const options = {
+    'width': 150,
+    'minHeight': 30,
+    'custom': false
+};
+/**
+ * @classdesc
+ * Class for tooltip, a tooltip used for showing some useful infomation attached to geometries on the map.
+ * @category ui
+ * @extends ui.UIComponent
+ * @param {Object} options - options defined in [tooltip]{@link tooltip#options}
+ * @memberOf ui
+ */
+class ToolTip extends UIComponent {
+    // TODO:obtain class in super
+    _getClassName() {
+        return 'ToolTip';
+    }
+    constructor(info) {
+        super(options);
+        if (isString(info)) {
+            this._content = info;
+        }
+    }
+    /**
+   * set the toolTip to a geometry or a map,and it needs some infomation to show
+   * @param {Geometry} owner - geometry to addto.
+   * @param {String} owner - information  to show in toolTip.
+   * @returns {UIComponent} this
+    **/
+    addTo(geometry) {
+        this._addTo(geometry);
+    }
+    /**
+   * Adds the UI Component to a geometry or a map
+   * @param {Geometry|Map} owner - geometry or map to addto.
+   * @returns {UIComponent} this
+   * @fires UIComponent#add
+   */
+    _addTo(owner) {
+        if (owner instanceof Geometry) {
+            owner._tooltip = this;
+            owner.on('mouseover', this.onMouseOver.bind(this));
+            owner.on('mouseout', this.onMouseOut.bind(this));
+        }
+        return super.addTo(owner);
+    }
+
+    /**
+    * get the UI Component's content
+    * @returns {String} tooltip's content
+    */
+    getContent() {
+        return this._content;
+    }
+
+    buildOn() {
+        const dom = createEl('div');
+        dom.className = 'maptalks-msgBox';
+        dom.id = 'tipDiv';
+        dom.style.width = options.width + 'px';
+        const content = '<div class="maptalks-msgContent">' + this. _content + '</div>';
```

Comment:
const content = '<div>${this. _content}</div>';
I changed like this,but it dosen't work.